### DISCUSSION TITLE
GeoMap : How to use a custom file for a geoJSON Layer (using docker image)

### DISCUSSION THREAD

**Discussion by fvisticot** [grafana/grafana#42853] [General]
I need to use a custom geoJSON file / layer.
I can choose an available file from the combo box (3 files by default)

BUT how can I define my own geoJSON file ? it seems not possible to provide an URL location or S3 bucket by example ?
Is it possible to start the docker image with dedicated option specifying the geoJSON file URL  ?

![image](https://user-images.githubusercontent.com/153679/145101648-58be1435-cc83-46a1-81ab-9eaf4516ffd5.png)

**Comment by yosiasz** [comment]
a year late but in case it helps someone else @fvisticot 

https://github.com/grafana/worldmap-panel/issues/6#issuecomment-1159118634

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
